gslb/ingestion/fullsync.go

```diff
@@ -173,6 +210,12 @@ func bootupSync(ctrlList []*GSLBMemberController, gsCache *avicache.AviCache) {
 		panic(err.Error())
 	}
 
+	err = checkGslbHostRulesAndInitialize()
+	if err != nil {
```

Comment:
why should panic if you were not able to fetch the GSLBHostRule? It could be missing, because it does not exist at all?

Reply:
If they are missing, there will not be an error, we will get the count as 0. This particular error indicates other errors like connectivity issues, informer errors. @sudswasavi